SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -6,9 +6,9 @@
 %define ne_data_dir %{_datadir}/%{ne_name}
 %define ne_include_dir %{_includedir}/%{ne_name}
 %define ne_sysconf_dir %{_sysconfdir}/%{ne_name}
-%define ne_log_dir /var/log/%{ne_name}
+%define ne_log_dir %{_localstatedir}/log/%{ne_name}
```

Comment:
I m wondering about the failing status of the testing. Could this be the offending change?

Reply:
Most likely not, the SPEC file is not exercised by the integration tests - looking into what is causing this.